modules/scatteredBidAdapter.js

```diff
@@ -0,0 +1,97 @@
+// jshint esversion: 6, es3: false, node: true
+'use strict';
+
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { deepAccess, logInfo, parseSizesInput } from '../src/utils.js';
+import { ortbConverter } from '../libraries/ortbConverter/converter.js';
+
+const BIDDER_CODE = 'scattered';
+const GVLID = 1179;
+export function buildImps(validBidRequests) {
```

Comment:
I think I forgot to remove this in my patch, it doesn't look to be used by anything now.

Reply:
Done!